backend/app/agent_types/constants.py

```diff
@@ -0,0 +1,2 @@
+FINISH_NODE_KEY = "finish"
```

Comment:
Added this to all cognitive architectures to have a dedicated node that signifies the end of graph workflow. I know that `END` exists but doing `as_node=END` lead to an error that stated that `END` wasn't part of the nodes.